modules/magniteAnalyticsAdapter.js

```diff
@@ -0,0 +1,851 @@
+import { generateUUID, mergeDeep, deepAccess, parseUrl, logError, pick, isEmpty, logWarn, debugTurnedOn, parseQS, getWindowLocation, isAdUnitCodeMatchingSlot, isNumber, deepSetValue, deepClone, logInfo, isGptPubadsDefined } from '../src/utils.js';
+import adapter from '../src/AnalyticsAdapter.js';
```

Comment:
It looks that this file has changed location on latest master, it is now in `../libraries/analyticsAdapter/AnalyticsAdapter.js`

Reply:
of course I pulled latest from master and did not rerun test before pushing.

I am sorry I have failed you

Reply:
Failing is better than not doing at all. Failing is a synonym of learning. :)